cmd/akashctl/main.go

```diff
@@ -41,8 +40,25 @@ func main() {
 
 	// Add --chain-id to persistent flags and mark it required
 	root.PersistentFlags().String(flags.FlagChainID, "", "Chain ID of tendermint node")
+	root.PersistentFlags().String(flags.FlagNode, "", "The node address")
+	_ = viper.BindPFlag(flags.FlagNode, root.PersistentFlags().Lookup(flags.FlagNode))
```

Comment:
Why is this different than the others?

FWIW the `--node` param usually comes from `flags.GetCommands()` and `flags.PostCommands()` in `cosmos/cosmos-sdk/client/flags/flags.go`.  See `x/deployment/client/cli/{query,tx}.go`.

The `events` command is not wrapped in that; I think you should just add it in `events/cmd/root.go`.

```go
		c.Flags().String(FlagNode, "tcp://localhost:26657", "<host>:<port> to Tendermint RPC interface for this chain")
		viper.BindPFlag(FlagNode, c.Flags().Lookup(FlagNode))
```